concent_api/core/tests/utils.py

```diff
@@ -0,0 +1,59 @@
+from base64                 import b64encode
+
+from django.conf            import settings
+from django.test            import TestCase
+
+from golem_messages         import dump
+from golem_messages         import message
+import dateutil.parser
+
+from utils.testing_helpers  import generate_ecc_key_pair
+
+
+class ConcentIntegrationTestCase(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.PROVIDER_PRIVATE_KEY, self.PROVIDER_PUBLIC_KEY = generate_ecc_key_pair()
+        self.REQUESTOR_PRIVATE_KEY, self.REQUESTOR_PUBLIC_KEY = generate_ecc_key_pair()
```

Comment:
If these are now class variables there should not be named like constants. Use lower case.